lib/transactions.ts

```diff
@@ -106,8 +106,8 @@ export class TransactionSender {
       );
 
       if (!opts?.sequential) {
-        signedSequence.map((set) =>
-          Promise.all(
+        await Promise.all(signedSequence.map(async (set) =>
+           await Promise.all(
```

Comment:
Ideally we're sending the txn signature back while we wait, but don't know outside of https://github.com/metaDAOproject/meta-dao-frontend/pull/63 and https://github.com/metaDAOproject/meta-dao-frontend/pull/75/files as options to do so. This is an improvement for sure and we're likely going to merge it (once I see it in action shortly).

Just wanted to draw attention to a possible solution.